libs/features/creature/src/select-creature/select-creature.component.html

```diff
@@ -41,6 +41,7 @@
       <keira-highlightjs-wrapper [code]="selectService.query" />
     </form>
 
+    {{ selectService.pageOffset }}
```

Comment:
do we want to show this?

Reply:
Haha, damn. Good catch. No we don't need that ^^